clerk/users.go

```diff
@@ -249,6 +249,27 @@ func (s *UsersService) DisableMFA(userID string) (*UserDisableMFAResponse, error
 	return &response, nil
 }
 
+type OAuthProviderToken struct {
+	Object         string      `json:"object"`
+	Provider       string      `json:"provider"`
+	Token          string      `json:"token"`
+	PublicMetadata interface{} `json:"public_metadata"`
```

Comment:
I think the `json.RawMessage` type communicates intent better than the `interface{}` type.
```suggestion
	PublicMetadata json.RawMessage `json:"public_metadata"`
```